Makefile

```diff
@@ -23,15 +23,16 @@ VALIDATE=docker run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local ${
 
 GOLANGCI_LINT=golangci-lint
 
-all: pull fetch patch clean gen mod docs move-other patch-post fmt test stage
+#all: pull fetch patch clean gen mod docs move-other patch-post fmt test stage
+all: pull patch clean gen mod docs move-other patch-post fmt test stage
 
 pull:
 	docker pull ${IMAGE}
 
-fetch:
-	curl \
-		-H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36' \
-		-o ${SPEC_FETCHED_FILE} ${SPEC_URL}
+#fetch:
```

Comment:
nit: We discussed previously that `fetch` does not need to be commented out since it was disabled in the `all` task. This is not merge blocking, just pointing it out.